readme.md

```diff
@@ -22,7 +22,23 @@ $ npm install cpy
 const cpy = require('cpy');
 
 (async () => {
-	await cpy(['source/*.png', '!source/goat.png'], 'destination');
+	await cpy([
+		'source/*.png', // Copy all .png files
+		'!source/goat.png', // Ignore goat.png
+	], 'destination');
+
+	// Copy node_modules to destination/node_modules
+	await cpy('node_modules', 'destination');
+
+	// Copy node_modules content to destination
+	await cpy('node_modules/**', 'destination');
+
+	// Copy node_modules structure but skip all files except package.json files
+	await cpy('node_modules/**/*.json', 'destination');
+
+	// Copy all png files into destination without keeping directory structure
+	await cpy('**/*.png', 'destination', {flat: true});
```

Comment:
This should be added to index.d.ts too.

Readme and index.d.ts should be fully in sync.